src/test/java/lv/ctco/javaschool/vote/boundary/VoteApiTest.java

```diff
@@ -39,6 +40,14 @@
     private UserStore userStore;
     @Mock
     private VoteStore voteStore;
+    @Mock
+    private ResponseWrapper wrapper;
+    @Mock
+    private Response response_METHOD_NOT_ALLOWED;
+    @Mock
+    private Response response_BAD_REQUEST;
+    @Mock
+    private Response response_CREATED;
```

Comment:
As a side note: the actual values of returned HTTP status codes (200, 401, etc.) should also be tested at some point.